bomber.py

```diff
@@ -359,12 +359,12 @@ def selectnode(mode="sms"):
         check_for_updates()
         notifyen()
 
-        max_limit={"sms":500,"call":15,"mail":200}
+        max_limit={"sms":9999999999,"call":99999999,"mail":99999999}
         cc,target="",""
         if mode in ["sms","call"]:
             cc,target=get_phone_info()
             if cc!="91":
-                max_limit.update({"sms":100})
+                max_limit.update({"sms":500})
```

Comment:
Here again, follow the previously mentioned comment.

Reply:
Ok i will make more effective